qemu/tests/cfg/qmp_event_notification.cfg

```diff
@@ -91,3 +91,8 @@
                 - qmp_none:
                     extra_params += " -watchdog i6300esb -watchdog-action none "
                     action_check = "none"
+        - qmp_guest_panicked:
```

Comment:
For pc and q35, pvpanic device is must for this case, so i suggest to add pvpanic device here even if this device is added by default in machine.cfg.  In this way , no dependence issue in the future.

Reply:
@yanan-fu updated and please review again?

 (17/18) smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_guest_panicked.from_guest: PASS (107.09 s)

Thanks
Jing